abs_april_tag_loc:
  {
    '1x':0,
    '1y':0,

    '0x':100,
    '0y':0,

    '10x':200,
    '10y':0,

    '9x':300,
    '9y':0,

    '3x':100,
    '3y':100,

    '4x':0,
    '4y':100,

    '21x':200,
    '21y':100,

    '11x':300,
    '11y':100,

    '5x':0,
    '5y':200,

    '6x':100,
    '6y':200,

    '14x':200,
    '14y':200,

    '13x':300,
    '13y':200,

    '8x':0,
    '8y':300,

    '7x':100,
    '7y':300,

    '15x':200,
    '15y':300,

    '16x':300,
    '16y':300
  }
